package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "setup-webpack-boiler-plate",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "@babel/core": "^7.21.4",
+    "@babel/preset-env": "^7.21.4",
+    "babel-loader": "^9.1.2",
+    "webpack-cli": "^5.0.1",
+    "webpack-dev-server": "^4.13.2"
```

Comment:
-  Please add webpack to your package.json because otherwise the npm start won't work 
![image](https://user-images.githubusercontent.com/62705947/230119409-d8398c57-d03c-43b8-b237-9f285ab7fb4f.png)
![image](https://user-images.githubusercontent.com/62705947/230119512-dc99c3f3-b14e-4e98-abff-c9ad65614887.png)